src/extension/extension.ts

```diff
@@ -353,6 +356,27 @@ export class RunmeExtension {
       context.subscriptions.push(new CloudAuthProvider(context))
     }
 
+    kernel
+      .runProgram('echo $0')
+      .then(({ output }) => {
+        const supportedShells = ['zsh', 'bash']
+        const isSupported = supportedShells.some((sh) => output?.includes(sh))
+
+        if (!isSupported) {
+          const showMore = 'Show more'
+          return window.showWarningMessage('Unsupported shell', showMore).then((answer) => {
+            if (answer === showMore) {
+              const dashboardUri = getFaqUrl('supported-shells')
+              const uri = Uri.parse(dashboardUri)
+              return env.openExternal(uri)
+            }
+          })
+        }
+      })
+      .catch((_e) => {
+        getLogger().warn('An error occurred while verifying the supported shell')
```

Comment:
If we can 100% make sure you have a compatible shell, shouldn't that be a reason to error/warn about the shell?